packages/@okta/vuepress-site/docs/guides/set-up-token-exchange/main/index.md

```diff
@@ -0,0 +1,345 @@
+
+#### Create custom scopes
+
+Add some custom scopes to both custom authorization servers to request during the token exchange calls.
+
+> **Note:** You can't include the `offline_access` scope or any of the OpenID Connect scopes when using a service app. This means that you can't request refresh tokens or ID tokens in a service app-initiated token exchange flow.
+
+1. From the Admin Console, select **Security** > **API**, and then select one of the authorization servers that you want to use.
+1. Select the **Scopes** tab, and then click **Add Scope**.
+1. In the dialog that appears, enter a scope name (**api:access:read**).
+1. Click Create.
+1. Repeat steps 1-4 and create a second custom scope called **api:access:write**.
+1. Repeat steps 1-5 for the second authorization server that you want to use.
+
+#### Update audience
+
+Just for this example, update the audience of the authorization servers to help with understanding the token exchange flow when you view the responses.
+
+1. On the **API** page, select the edit (pencil) icon to the right of one of the authorization servers.
+1. In the **Settings** section, click **Edit**.
+1. Change the **Audience** to **com.api.atko** and click **Save**.
+1. Repeat steps 1-3 for the second authorization server.
+
+#### Create access policies and rules
+
+An [access policy](/docs/guides/configure-access-policy/main/) helps you secure your APIs by defining different access token lifetimes for a given combination of the grant type, user, and scope. You create policy rules to determine if an application should be permitted to access specific information from your protected APIs and for how long. Access policies are specific to a particular authorization server, the client applications that you designate for the policy, and the API resource for which you want to mint tokens.
+
+1. From the **API** page, select the edit (pencil) icon for the authorization server that you want to use for the initial authentication.
+1. Select the **Access Policies** tab, and then click **Add New Access Policy** to add a policy that allows the native app to access API1.
+1. In the **Add Policy** dialog that appears, enter the following:
+
+   * **Name:** Enter a name for the new access policy: **Access API1**.
+   * **Description:** Enter a description.
+   * **Assign to:** Select **The following clients**, start typing the name of the native app that you created earlier, and select it from the list that appears.
+
+1. Click **Create Policy**.
+1. Click **Add Rule** and in the dialog that appears, enter the following:
+
+   * **Name:** Enter a name for the rule: **Mobile app to API1**.
+   * **AND Scopes requested:** Select **The following scopes** and enter **openid**.
+
+1. Click **Create rule**.
+1. Repeat steps 1-5 to create a policy and a rule that allows the service app that represents API1 to talk to API2. Use the following values for the policy:
+
+   * **Name:** **Access API2**.
```

Comment:
```suggestion
   * **Name:** Enter **Access API2**.
```

Reply:
Also, this one doesn't need a description like step 3?

Reply:
a description isn't a required field when you create a Rule.